.eslintrc.json

```diff
@@ -93,6 +93,31 @@
         "import/no-named-as-default-member": "off",
         "import/no-unresolved": "off"
       }
+    },
+    {
+      // Only enable rules that depend on type checking on TS files,
+      // to avoid a barrage of warnings for older code:
+      "files": ["**/*.{ts,tsx}"],
+      "excludedFiles": ["src/app/(nextjs_migration)/**/*"],
```

Comment:
Q: Why excluding this directory specifically?

Reply:
Heh, actually, I was going to say that there's lots of old copy-pasted code there from the pre-TS era and thus making it pass was too painful, but it was actually doable, so this exclude is now excluded: 68e51ef1e